src/vm/mscorlib.h

```diff
@@ -752,10 +752,6 @@ DEFINE_CLASS(PARAMETER_MODIFIER,    Reflection,             ParameterModifier)
 
 DEFINE_CLASS(POINTER,               Reflection,             Pointer)
 
-DEFINE_CLASS_U(Reflection, Pointer, ReflectionPointer)
```

Comment:
Why are you removing this? It is here to verify the layout is in sync between managed and unmanaged sides.